content/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups.md

```diff
@@ -104,7 +104,7 @@ You can update the access policy of a runner group, or rename a runner group.
 
```

Comment:
I think we need to also update `{% data reusables.github-actions.self-hosted-runner-configure-runner-group-access %}` above (in "Changing the access policy of a self-hosted runner group")?

Reply:
Ooh yes, thanks for spotting that one! 🚀